compiler/rustc_codegen_ssa/src/target_features.rs

```diff
@@ -179,6 +179,7 @@ const X86_ALLOWED_FEATURES: &[(&str, Option<Symbol>)] = &[
     ("f16c", Some(sym::f16c_target_feature)),
     ("fma", None),
     ("fxsr", None),
+    ("gfni", Some(sym::avx512_target_feature)),
```

Comment:
It's honestly kind of weird that this isn't prefixed `v` for `vgfni` like the other features.

Reply:
I think it's because the `v` prefix implies VEX- or EVEX-encoded instructions (e.g. `VGF2P8AFFINEQB`), but GFNI also introduces plain SSE instructions (e.g. `GF2P8AFFINEQB`).  This is the same distinction as PCLMULQDQ vs VPCLMULQDQ or AES vs VAES for example, but the difference there is the different encodings have separate CPUID flags, whereas GFNI has just one.